test/specs/services/conversation.spec.js

```diff
@@ -240,7 +240,20 @@ describe('Conversation service', () => {
 
         conversationStartedSuite(conversationService.sendMessage('message'));
 
-        it('should add message and send message', () => {
+        it('should add message and send message', function*() {
+            mockedStore = createMockedStore(sandbox, getProps());
+
+            yield mockedStore.dispatch(conversationService.sendMessage('message'));
```

Comment:
We are? We're checking that `postSendMessage` is called - one for each message.
Unless you mean check the args, too? I can do that.